### DISCUSSION TITLE
Why is the light still bright when IBL / Lights is turned off?

### DISCUSSION THREAD

**Discussion by cx20** [Reon90/redcube#31] [Q&A] [answered]
I tried to display [AlphaBlendModeTest.gltf](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/AlphaBlendModeTest) in RedCube.js.
However, I did notice that even with both `IBL` / `Lights` turned off, the middle of the image was still displayed brightly.

Is this a matter of usage? I had assumed that if I turned off both `IBL` / `Lights`, the whole thing would be dark.

Current result:
|IBL|Lights|[RedCube + AlphaBlendModeTest.gltf result](https://cx20.github.io/gltf-test/examples/redcube/index.html?category=tutorialModels&model=AlphaBlendModeTest&scale=0.5&type=glTF)|
|:---:|:---:|:---:|
|off|off|![image](https://user-images.githubusercontent.com/1833979/108578637-8a5ad880-7366-11eb-9c5d-53d9d845975b.png)|
|off|on|![image](https://user-images.githubusercontent.com/1833979/108578644-9050b980-7366-11eb-8c44-b38aa090da3b.png)|
|on|off|![image](https://user-images.githubusercontent.com/1833979/108578652-95ae0400-7366-11eb-9fd2-e9482e7cabb7.png)|
|on|on|![image](https://user-images.githubusercontent.com/1833979/108578657-9b0b4e80-7366-11eb-91e6-a4b9e54a7d70.png)|

**Comment by Reon90** [comment, marked as answer]
@cx20 Actually, it's an incredible interesting bug, this white color is a background of your index.html.
I couldn't reproduce it, because my page has `background: #000;`

**Comment by cx20** [comment]
@Reon90 Thanks! As you advised, changing the background color to `#000` solved the problem.

![image](https://user-images.githubusercontent.com/1833979/108638053-af6b5a80-74d0-11eb-9507-78b458579fe6.png)

**Reply by Reon90** [reply]
@cx20 fixed in 2.5.12
